sdk/core/Azure.Core/tests/TestFramework/JsonStream.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace Azure.Core.Testing
+{
+    public class JsonStream : IDisposable
+    {
+        private readonly MemoryStream _buffer = new MemoryStream();
+
+        public string Content => Encoding.UTF8.GetString(_buffer.GetBuffer(), 0, (int)_buffer.Length);
+
+        public Utf8JsonReader CreateReader(JsonReaderOptions options = default) => new Utf8JsonReader(_buffer.GetBuffer(), options);
```

Comment:
This method is never used.

Reply:
I had planned to beef it up eventually, but can move it out for now if you'd like. Figured it'd be a good helper class.